### DISCUSSION TITLE
2006/08/01/%E3%80%8A%E7%AA%81%E7%A0%B4%E5%81%9C%E6%BB%9E%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#101] [Announcements]
# 2006/08/01/%E3%80%8A%E7%AA%81%E7%A0%B4%E5%81%9C%E6%BB%9E%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

《致加西亚的一封信》被很多人誉为洗脑书，意志不坚强的员工看过之后就会不问报酬，尽心竭力为公司卖命。所以，为了不被洗脑，每个员工都应当拓宽阅读面，兼听则明嘛。

https://blog.qizhen.xyz/2006/08/01/%E3%80%8A%E7%AA%81%E7%A0%B4%E5%81%9C%E6%BB%9E%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

捷周
2007年01月3日 13:55 编辑
楼主言之有理，不过个人觉得５年太久了．

回复
weili
2006年08月17日 23:03 编辑
这么多有意思的东西，很佩服你的坚持劲，居然把这么多东西写得这么透彻~

回复
竹贤
2006年08月17日 22:23 编辑
每个人的能力、特长、追求等都不同，所以不可能有一条适合所有人的标准的道路。关键是能否找到一个最适合自己的位子，这既有个人的能力的条件，更有机遇的限制。个人努力当然是必不可少的，但很多时候往往是可遇而不可求。
升职与成功不能划等号；丰厚的酬金是重要前提，但不是唯一的标准；关键是你个人的感觉和心态。职务的分布总是金字塔形的，在底层的总是最多数，如果人人升迁，成了倒金字塔形，能站得住吗？

回复
yoyo
2006年08月2日 19:12 编辑
你什么时候跳？
